packages/wrapped-keys-lit-actions/src/lib/raw-action-functions/solana/signTransactionWithEncryptedSolanaKey.ts

```diff
@@ -1,22 +1,22 @@
-const {
-  getDecryptedKeyToSingleNode,
-} = require('../../internal/common/getDecryptedKeyToSingleNode');
-const {
+import { getDecryptedKeyToSingleNode } from '../../internal/common/getDecryptedKeyToSingleNode';
+import {
   signTransactionSolanaKey,
   validateUnsignedTransaction,
-} = require('../../internal/solana/signTransaction');
+} from '../../internal/solana/signTransaction';
+
+interface SignTransactionWithEncryptedSolanaKeyParams {
+  accessControlConditions: any; // Define a more specific type if possible
+  ciphertext: string; // The encrypted Wrapped Key
+  dataToEncryptHash: string; // The hash of the data to encrypt
+  unsignedTransaction: any; // Define a more specific type if possible
```

Comment:
Same as with the ethereum one, there is a definition for this in `signTransactionSolanaKey`